### ISSUE TITLE
Minifying in parallel

### ISSUE THREAD

**Issue by andrey81inmd** [davidB/yuicompressor-maven-plugin#55]
It would be nice to have a feature to minify multiple files in parallel. If I have 200 js files to minify in a build, it takes a good several minutes. It should be possible to fork X number of processes to minify different files at the same time, so that the whole build would be done quicker.

Thanks,
Andrey
